docs/docsite/rst/dev_guide/testing_integration.rst

```diff
@@ -132,14 +136,18 @@ the Ansible continuous integration (CI) system is recommended.
 Running Integration Tests
 -------------------------
 
-To run all CI integration test targets for POSIX platforms in a Ubuntu 16.04 container::
+To run all CI integration test targets for POSIX platforms in a Ubuntu 18.04 container::
 
-    ansible-test integration --docker ubuntu1604 -v shippable/
+    ansible-test integration shippable/ --docker ubuntu1804
 
 You can also run specific tests or select a different Linux distribution.
-For example, to run tests for the ``ping`` module on a Ubuntu 14.04 container::
+For example, to run tests for the ``ping`` module on a Ubuntu 18.04 container::
+
+    ansible-test integration ping --docker ubuntu1804
+
+Use the ``--docker-no-pull`` option to avoid pulling the latest container image every time when you run the tests::
```

Comment:
This is really only useful when using custom images. All of the built-in images are specified by version or SHA, so skipping the pull results in either an error (image not found) or no change (image at specified version or SHA already present).

Reply:
ok, thanks for the expanation, I removed the example